dts/bindings/mdio/xilinx,axi-ethernet-7.-2-mdio.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for MDIO controllers
+
+include: mdio-controller.yaml
+
+compatible: xlnx,axi-ethernet-7.2-mdio
+
+description: MDIO interface of Xilinx AXI 1G/2.5G Ethernet Subsystem.
+
+properties:
+  reg:
+    required: true
+    type: array
+    description: |
+      Control registers of the MDIO device. Should be the same value that is also
```

Comment:
maybe there should be a parent node with the reg addr if they will be the same?

Reply:
Moved it to a parent node with compatible and regs property.